src/app/static/src/js/components/runWorkflow/runWorkflowCreate.vue

```diff
@@ -103,7 +103,7 @@
             },
             clone: function () {
                 if (this.selectedWorkflow && this.runWorkflowMetadata) {
-                    const {reports, git_branch, git_commit} = this.runWorkflowMetadata
+                    const {reports, git_branch, git_commit} = workflowRunDetailsToMetadata(this.runWorkflowMetadata)
```

Comment:
`Clone` should copy every everything apart from name shouldn't it (i.e. should copy instances as well)? So I think this clause can be replaced with. 
```
  const clonedWorkflow = {
    ...workflowRunDetailsToMetadata(this.runWorkflowMetadata),
    name: ""
  }; 
  this.$emit("clone", clonedWorkflow); 
```